pmp-reactjs-ui/src/pages/Dashboard.js

```diff
@@ -4,8 +4,8 @@ function Dashboard() {
 
   const navigate = useNavigate();
 
-  const partnerCertificate = () => {
-    navigate('/partnermanagement/partnerCertificate')
+  const PartnerCertificatesList = () => {
+    navigate('/partnermanagement/PartnerCertificatesList')
   };
```

Comment:
it should be camelCase only 
all variables and methodnames

Reply:
Done.